copycat/api/src/main/java/org/apache/kafka/copycat/sink/SinkTaskContext.java

```diff
@@ -59,6 +63,40 @@ public void offset(Map<TopicPartition, Long> offsets) {
     }
 
     /**
+     * Set the backoff timeout. SinkTasks should use this to indicate that they need to retry certain
+     * operations after the backoff timeout.
+     * @param backoffMs the backoff timeout in milliseconds.
+     */
+    public void backoffMs(long backoffMs) {
+        this.backoffMs = backoffMs;
+    }
+
+    /**
+     * Get the backoff in milliseconds set by SinkTasks. Used by the Copycat framework.
+     * @return the backoff timeout in milliseconds.
+     */
+    public long backoffMs() {
```

Comment:
I'm noticing that we're now including quite a few things in this interface that are really internal details -- the methods that set these values are intended to be public APIs called by SinkTasks, but we also expose methods to get those values back out so the framework can get at them. Should we split this between an interface/abstract class `SinkTaskContext` which is in `copycat/api` and an implementation in `copycat/runtime`?